src/statd/statd.c

```diff
@@ -145,11 +139,25 @@ static json_t *json_get_ip_link(char *ifname)
 	return j_root;
 }
 
+static const char *get_yang_operstate(const char *operstate)
+{
+	if (strcmp(operstate, "DOWN") == 0)
+		return "down";
+	if (strcmp(operstate, "UP") == 0)
+		return "up";
+	if (strcmp(operstate, "LOWERLAYERDOWN") == 0)
+		return "lower-layer-down";
+	if (strcmp(operstate, "NOTPRESENT") == 0)
+		return "not-present";
+
+	return "unknown";
```

Comment:
This fixes the issue I saw with statd crashing on unknown strings, but is a regression to the `str_to_lowercase()` in that it maps `dormant` and `testing` to `unknown`.